tools/proofers/previewControl.js

```diff
@@ -99,6 +99,13 @@ function initPrev() {
                 : "pre"
         );
         prevWin.innerHTML = preview.txtout;
+        if (preview.ok) {
+            try {
```

Comment:
Rather than load the MathJax JS file for every proofreading page -- may of which won't ever use Format Preview -- consider only loading the JS when needed, possibly with https://api.jquery.com/jQuery.getScript/. I also see merit in having this be an option in Format Preview so that we only attempt loading the JS and running MathJax if the user wants to preview math.

Reply:
Good point, I'll do that.